app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -57,20 +68,26 @@ class DeadlineDetailsActivity : AppCompatActivity() {
     private lateinit var doneButton: CheckBox
     private lateinit var notificationView: TextView
     private lateinit var descriptionView: EditText
+    private lateinit var downloadLinkText: TextView
     private lateinit var locationView: TextView
 
     // Set them on default value, waiting the fetch of the deadlines
     private var notificationSelected: BooleanArray = booleanArrayOf(false, false, false, false)
     private var dateTime: LocalDateTime = LocalDateTime.of(2022, 10, 10, 10, 10)
     private var state: DeadlineState = DeadlineState.TODO
+    private var pdfLink: String = ""
 
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_deadline_details)
 
+        Firebase.initialize(this)
+        firebasePdfRepository = FirebasePdfRepository(FirebaseStorage.getInstance())
```

Comment:
Use dependency injection instead of manually instantiating the repository, otherwise we cannot test it